packages/twenty-server/src/modules/messaging/message-import-manager/listeners/messaging-import-manager-message-channel.listener.ts

```diff
@@ -30,4 +36,25 @@ export class MessagingMessageImportManagerMessageChannelListener {
       },
     );
   }
+
+  @OnEvent('messageChannel.updated')
+  async handleUpdatedEvent(
+    payload: ObjectRecordUpdateEvent<MessageChannelWorkspaceEntity>,
+  ) {
+    if (
+      objectRecordChangedProperties(
+        payload.properties.before,
+        payload.properties.after,
+      ).includes('isContactAutoCreationEnabled') &&
+      payload.properties.after.isContactAutoCreationEnabled
+    ) {
+      await this.messageQueueService.add<MessagingCreateCompanyAndContactAfterSyncJobData>(
```

Comment:
What about contactCreationQueue, is it still used? Just wandering since you are pushing to the messagingQueue queue here